src/kvirc/sparser/KviIrcServerParser_ctcp.cpp

```diff
@@ -826,6 +826,17 @@ void KviIrcServerParser::parseCtcpRequest(KviCtcpMessage * msg)
 {
 	msg->pData = extractCtcpParameter(msg->pData, msg->szTag);
 
+	bool bAction = KviQString::equalCI(msg->szTag, "ACTION");
+
+	if (!bAction)
```

Comment:
I put it before that so that self-DCC messages wouldn't give 'Ignoring DCC from [self]' messages.